guides/extensions/rest.md

```diff
@@ -31,7 +31,7 @@ You can do this directly on your machine.
 After installing and confirming that Libindy is installed, simply run:
 
 ```sh
-npx -p @aries-framework/rest afj-rest start \
+npx -p @credo-ts/rest afj-rest start \
```

Comment:
```suggestion
npx -p @credo-ts/rest afj-rest start \
```

is/should the `afj-rest` part correct?

Reply:
I'm going to update this to credo-rest. Good catch